cmd/skaffold/app/cmd/flags.go

```diff
@@ -573,6 +564,15 @@ var flagRegistry = []Flag{
 		},
 		IsEnum: true,
 	},
+	{
+		Name: "hydration-dir",
```

Comment:
When would a user be expected to set this flag?  If it's a _just in case_ maybe we should mark it as `Hidden: true` so that it's available but not visible.

Reply:
I'm expecting users to use the `--hydration-dir` (some sort of structured configurations) more than `--output` (a flattened file).